auth-api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>2.0.0-ALPHA-2-SNAPSHOT</version>
+    <version>2.0.0-ALPHA-3-SNAPSHOT</version>
```

Comment:
Is this intentional, or a side-effect of re-doing 2.0.0-ALPHA-1 release @jeffreyscarpenter ?